react-frontend/src/components/Home/bcLivesteam.js

```diff
@@ -0,0 +1,133 @@
+import React, { useContext } from 'react';
+import dateFormat from 'dateformat';
+import { Col, Row } from 'react-flexbox-grid';
+import { PageContainer } from '../StyleComponents/pageContent';
+import { Heading } from '../StyleComponents/headings';
+import Query from '../Query';
+import { LinkExternalButton } from '../StyleComponents/htmlTags';
+
+import LIVESTREAM_QUERY from '../../queries/digitalBCLivestream/livestreams';
+import { AppConfigContext } from '../../providers/AppConfig';
+const youTubeLogo = require('../../images/icons/yt_logo_rgb_light.png').default;
+
+const BCLivestream = () => {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <Query query={LIVESTREAM_QUERY}>
+      {({ data: { digitalLivestreamContact, digitalBcLivestreams } }) => {
+        const imageSource =
+          strapiMediaUrl + digitalBcLivestreams[0]?.EventImage?.url;
+        // if no next event is provided in strapi, this section will not render
+        const now = new Date();
+        const nextEventTime = new Date(digitalBcLivestreams[0]?.BroadcastTime);
+
+        {
+          /* Only render if link provided */
+        }
+        const pastBroadcastLink = digitalLivestreamContact?.PastEpisodeURL && (
+          <div>
```

Comment:
Sorry to be a pain. Could you add `style={{ marginTop: '16px' }}` to this `<div>` -- to add the desired spacing for the scenario where the upcoming broadcast description is omitted?

![Screen Shot 2021-09-23 at 1 44 27 PM](https://user-images.githubusercontent.com/16946297/134582046-bc3195e9-9097-45c6-a9fb-969c8d6d21fb.png)